### DISCUSSION TITLE
ILI9341 with S3 hookup

### DISCUSSION THREAD

**Discussion by builderdev212** [Bodmer/TFT_eSPI#2581] [Q&A - General]
Howdy folks. I've been designing a custom ESP32-S3 board and plan on using [this display from digikey](https://www.digikey.com/en/products/detail/dlc-display-co-ltd/DLC0283BDP04DF-4/13624810). I've never worked with displays with the ESP32-S3 before, is there any documentation as to how I should hook the display up?

**Comment by Bodmer** [comment]
Here is an example GPIO allocation:

https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup70b_ESP32_S3_ILI9341.h
